chapters/en/slides/module6/module6_01.Rmd

```diff
@@ -0,0 +1,340 @@
+which is something that the reader might already have in mind since the previous slide.
+Let's explore this next.
+
+---
+
+## Demand for beef is the biggest reason for deforestation in Brazil
+
+```{python}
+# TODO save wrangled data
+brazil_drivers = pd.read_csv('data/drivers-forest-loss-brazil-amazon.csv').query('Year > 2003')
+brazil_drivers = brazil_drivers.melt(id_vars=['Entity', 'Code', 'Year'], var_name='type', value_name='deforestation_hectares').groupby('type').sum().reset_index()
+alt.Chart(brazil_drivers).mark_bar().encode(
+    x='deforestation_hectares',
+    y=alt.Y('type', title='', sort='x'))
+```
+
+Notes:
+This slide shows the biggest drivers of deforestation in Brazil
+between the years 2003 and 2013.
+
+We can see that loss of forest 
+is dominated by the demand for beef,
+which leads to permanent clearing of forest
+to create pastures for cattle grazing.
+
+There is also a smaller contributions from small scale clearings,
+fires, and commercial crops.
+
+Although beef is the major culprit in rain forest deforestation,
+it could be the case that our demand for more food has simply increased
+and cattle herds are an effective use of land to create food for human consumption.
+Let's flip to the next slide to find out!
+
+---
+
+## Beef is highly inefficient use of land to generate calories
+
+```{python}
+# TODO save wrangled data
+land_use = pd.read_csv('data/land-use-kcal-poore.csv').rename(columns={'Land use per 1000kcal (Poore & Nemecek, 2018)': 'Land use per 1000kcal (m2)'}).query('Entity in ["Barley", "Maize", "Rice", "Tofu (soybeans)", "Cassava", "Bananas", "Eggs", "Poultry Meat", "Dark Chocolate", "Milk", "Cheese", "Coffee", "Lamb & Mutton", "Beef (beef herd)"]')
+alt.Chart(land_use).mark_bar().encode(
+    alt.X('Land use per 1000kcal (m2)'),
+    alt.Y('Entity', sort='x', title=''))
+```
+
+Notes:
+The plot in this slide reveals that herding cattle for the purpose of beef production
+is one of the most inefficient ways to produce food for human consumption.
+
+Together with the large demand for beef products
+this inefficient use of land creates an unfortunate situation in Brazil
+where a lot of tropical forest is being cut down to make room for new pastures.
+
+When telling a story,
+it is important to not jump to conclusions
+or suggest that solutions to problems are simpler than what they might really be.
+
+So far we have seen that beef is a major contributing factor to deforestation in Brazil,
+but there are many more aspect of the food production chain that we have not considered,
+such as the use of other resources
+
+Changing beef for other products would be highly beneficial,
+but how achievable it is 
+and what cost is associated with making the change
+we don't know from the data we have explored here.
+
+Let's continue our story by looking at the same breakdown for Indonesia,
+is beef production the biggest issue there as well?
+
```

Comment:
```suggestion

If we find evidence to support our hypothesis that global beef demand is a
major driver of deforestation in Brazil, then we might want to consider what interventions
could reverse or slow this effect.

The plot in this slide reveals that raising cattle for the purpose of beef production
is one of the most inefficient ways to produce food for human consumption. Armed 
with this knowledge (and the data to back it up), we could then suggest that one
way to decrease deforestation in Brazil may be to decrease the global demand for beef,
while increasing the demand for other protein-rich foods that can be grown in Brazil
in a more efficient way. 

However, when telling a story, it is important to not jump to conclusions or suggest that 
solutions to problems are simpler than what they might really be. So far we have seen that 
beef may be a major contributing factor to deforestation in Brazil, but there are many others 
aspects of the food production chain that we have not considered.  Additionally, decreasing 
the global demand for beef would be a challenging task, and would likely require the cooperation 
of many countries to implement education campaigns, likely along with other measures. In
telling this story, it would be important to also include these uncertainties and potential challenges.

Let’s continue our story, and next ask whether the large deforestation in Indonesia may be
also due to clearing land for pasture.
```

Reply:
I was thinking about using protein per m2 instead of kcal, which might be a more fair comparison. Or I don't know, I think both matters and it is best to show both but not doing that here for now. I'll keep it as is for now but fix at the end if I have time.